pytest_inmanta/plugin.py

```diff
@@ -138,6 +138,10 @@ def project(project_shared, capsys):
 
 @pytest.fixture()
 def project_no_plugins(project_shared_no_plugins, capsys):
+    LOGGER.warning(
+        "The project_no_plugins fixture is deprecated in favor of the %s environment variable.",
+        option_to_env["inm_no_load_plugins"],
+    )
```

Comment:
This is just a normal log message and not a deprecation warning? If I am not mistaken, deprecationwarning should be logged via the `warnings.warn()` method. Probably your test case is catching another deprecationwarning than this one.

Reply:
You're right. I started out wanting to raise a `DeprecationWarning` but then noticed the logger and decided to use that instead. I actually have little experience here? Should all warnings use the logger or should I raise an actual `DeprecationWarning`?

Reply:
It seems that the deprecationwarning is ignored by python by default, so to me it looks better to use the standard logger.

Reply:
I think pytest explicitly warns about deprecation warnings, which is what counts here. I've just discovered that the logger's warning is actually ignored by pytest, so I guess I'll go with the `DeprecationWarning` then.